src/main/resources/assets/api-docs.yaml

```diff
@@ -488,6 +488,36 @@ paths:
             application/json:
               schema:
                 $ref: './schemas/DataAccessRequest.yaml'
+  /api/dar/progress_report/{parentReferenceId}:
```

Comment:
The API location as defined in `DataAccessRequestResourceVersion2` is actually `/api/dar/v2/progress_report/{parentReferenceId}`